akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -162,23 +165,16 @@ abstract class ActorMaterializer extends Materializer {
    */
   def isShutdown: Boolean
 
-  /**
-   * INTERNAL API: this might become public later
-   */
-  private[akka] def actorOf(context: MaterializationContext, props: Props): ActorRef
+  def materialize[Mat](_runnableGraph: Graph[ClosedShape, Mat],
+                       subflowFuser: GraphInterpreterShell ⇒ ActorRef): Mat
```

Comment:
GraphInterpreterShell is still in internal `impl` package. I'm not sure what it would mean to move those to public package.

An alternative would be to define a new trait `ExtendedActorMaterializer` with these methods made public. It would then live in the `impl` package (for now) and you would downcast to that instead of `ActorMaterializer`

`ActorMaterializerImpl` extends `ExtendedActorMaterializer`
`ExtendedActorMaterializer` extends `ActorMaterializer`